### DISCUSSION TITLE
403 Client Error

### DISCUSSION THREAD

**Discussion by linkeins** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#335] [Q&A] [answered]
HTTP error while trying to fetch Udemy information
403 Client Error: Forbidden for url: https://www.udemy.com/api-2.0/users/me/subscribed-courses/?ordering=-last_accessed&fields%5Bcourse%5D=@min,enrollment_time,published_title,&fields%5Buser%5D=@min&page=1&page_size=100
Unable to fetch existing courses. Login was not successful

**Comment by cullzie** [comment, marked as answer]
This should be fixed now if you download the latest version from pypi
